### PR TITLE
Data from form dynamically submitted

### PR DESCRIPTION
**Pull request by LauraMihalca**

### CODE REVIEW

File: project/src/events/new-evaluation/new-evaluation.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    };
+
+    technicalLevel = document.querySelector('input[name="level"]:checked').value;
+
+    textarea = getTextarea();
+
+    fieldset = getFieldset();
+
+    const newEvaluationForm = new FormModel(inputData, technicalLevel, textarea, fieldset);
+    console.log(newEvaluationForm);
```
Comment (s-timofte) [inline project/src/events/new-evaluation/new-evaluation.js:55]: I think you forgot to remove this after the tests

File: project/src/helper/helper.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+function constructId(title) {
+    return title.split(/[ |,|]/).join('');
+}
+
+function constructIdFieldset(legend, label) {
+    const newLegend = legend.split(/[ |,|]/).join('');
```
Comment (s-timofte) [inline project/src/helper/helper.js:6]: Is there any already declared function you could use here?

**Review by s-timofte — APPROVED** [review]
Nice, I like the progress :)

File: project/src/events/new-evaluation/new-evaluation.js (hunk 1 above)
Comment (LauraMihalca) [reply project/src/events/new-evaluation/new-evaluation.js:55]: Oh,yes..

File: project/src/helper/helper.js (hunk 2 above)
Comment (LauraMihalca) [reply project/src/helper/helper.js:6]: 👍

**Pull request merged**